docs/pages/eslint_plugin.js

```diff
@@ -134,6 +134,10 @@ With AUTOFIX!
         With AUTOFIX!
       `}
         />
+        <MainSection.Subsection
+          title="gestalt/no-workflow-status-icon"
+          description="Prevent the usage of 'workflow-status-[...]' icons on `Icon` component where the values of props matches with the [Status](/status) component usage."
```

Comment:
Prevent the usage of 'workflow-status-[...]' icons on `Icon` component when the style matches one of the [Status](/status) types.

Reply:
@EduhCosta

Reply:
Fix